fusion_report.log.2024-06-25

```diff
@@ -0,0 +1,64 @@
+2024-06-25 09:55:25,106 - CRITICAL - fusion_report.common.net - table "mbca" already exists
```

Comment:
The PR looks good to me! Although, I am not sure I understand what this file is for.